script/output/dev-0/v1-deployment.json

```diff
@@ -1,8 +1,8 @@
 {
   "addresses": {
-    "queryClient": "0x9e81a7a0A234E6Ede4604DD7823018D0efE53b4b",
-    "queryClientProxy": "0x5C189dC46c2c6136dBaFdAAa02Db46CD2dF683A3",
-    "registryImpl": "0x7a9C8a9af98119249F1b8f4786Fa9aE1B5fBB60A",
+    "queryClient": "0x0000000000000000000000000000000000000000",
```

Comment:
This shouldn't be 0 - there is something wrong there.

Reply:
cc/ @RyanRHall

Reply:
This should be `0x400e7aAEadcc134B0d8413092119221Ad736ed3e` - I'm not sure why the script didn't automatically fill it in.